README.md

```diff
@@ -41,12 +41,14 @@ Then copy the list of game IDs and put the new values in all the relevant fields
 Load the application as an unpackaged app in Overwolf:
 
 -   Open Overwolf
--   Open settings
+-   Open Settings
 -   Open the `About` tab
 -   Click on `Development options`
 -   Click `Load unpacked extension...`
 -   Select the `dist` folder.
 
+PS: There is little to no reason to try and run the app, if you're not whitelisted by Overwolf, the app will not work. They block it at some base level. So unless you actively have developed on Overwolf before, you will not be able to test the app.
```

Comment:
Please add a more friendly variation of this sentence right below the `How to contribute` headline. Maybe

`To be able to run a development version, you need to be a developer that is allow-listed by Overwolf. See https://dev.overwolf.com/ow-native/reference/ow-sdk-introduction#get-whitelisted-as-a-developer for details.`

Reply:
Got ya